tests/SuiteCRM/Test/SuitePHPUnit_Framework_TestCase.php

```diff
@@ -88,6 +90,8 @@ public function tearDown()
         $GLOBALS['log'] = $this->log;
 
         DBManagerFactory::$instances = $this->dbManagerFactoryInstances;
+        
+        parent::tearDown();
```

Comment:
Why do we need to use the tearDown in the set up of this class? Should this call be here?

Reply:
Just in case. In long term, the PHP-Unit is a third party lib, I can not garanty it's never will change..

Reply:
Github was collapsed that code, cause me confusion, nevermind :+1: